jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java

```diff
@@ -216,15 +216,15 @@ void initialize() {
       } catch (UnknownHostException e) {
          // Let's just use a random hostname
          hostname = UUID.randomUUID().toString();
-         logger.info("Unknown host error: {}, using random", e);
+         logger.warn("Unknown host error: {}, using random", e);
```

Comment:
Should we try to standardize this and anytime we have a try/catch that throws an error log it as an error level?

Reply:
If an exception doesn't cause a serious failure, I think using `warn` is acceptable. Since the program can continue in this case, I used the `warn` level. In most cases we are using `error` in exception catch blocks, though.

Do you think we should transition to using `error` for all exception blocks by default & `critical` when the program cannot continue?

Reply:
If we didn't catch that exception it would cause a serious failure right? We're catching an exception, and error if you will. I think it makes sense to log error for any exception as the terms are somewhat synonymous. Critical makes sense if the app is going to fail.

Reply:
Sounds good, I'll make that change!

Reply:
I've modified the logging level being used in two exception catch blocks to `error` instead of `warn`.

Reply:
I've gone through our exception catch blocks and verified that we're using the `error` level whenever we print to a logger.